doc/DeveloperGuide.md

```diff
@@ -87,13 +88,37 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to list persons
+2. AddressBook shows a list of persons
+3. User requests to rename tag of all the people in the list.
+4. Addressbook will prompt for confirmation about the rename.
+5. User accepts the change.
+6. AddressBook renames the person's tag <br>
```

Comment:
Perhaps you might want to take note on formatting? `<br>` is not suppose to appear here =)